INSTALL.md

```diff
@@ -194,7 +188,121 @@ Run [NPM](https://www.npmjs.com/) to install all the required Node modules:
 npm install
 ```
 
-#### Setup Database
+## Manual setup on Fedora (29)
+
+### Prerequisites
+
+#### Ruby version
+
+The project supports **Ruby 2.3.x**.
+
+Verify you have a proper version by running on your terminal:
+```bash
+ruby -v
+```
+
+> You can use [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install ruby.
+
+#### Node version
+
+The project supports **Version: 8.X.X**.
+
+You might want to use [nvm](https://github.com/creationix/nvm/) to install and work with specific Node versions:
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bashm
+```
+
+Restart the terminal and install Node:
+
+```bash
+nvm install 8
+nvm use 8
+```
+
+#### Dependencies
+
+```shell
+sudo yum install patch autoconf automake bison libffi-devel libtool libyaml-devel readline-devel sqlite-devel zlib-devel openssl-devel ImageMagick ImageMagick-devel mysql-devel postgresql-devel chromedriver memcached
+
+sudo service memcached restart
+```
+
+#### Database (Postgres / MySQL / Oracle)
+
+Postgres, MySQL or Orcacle has to be running for the application to work. The easiest way to do it is in a [Docker](https://www.docker.com/) container by simply running:
+
+```shell
+docker run -d -p 5433:5432 -e POSTGRES_USER=postgres -e POSTGRES_DB=3scale_system_development --name postgres10 circleci/postgres:10.5-alpine
+```
+
+Alternatively you can run Redis directly on your machine by following [this article](https://www.if-not-true-then-false.com/2012/install-postgresql-on-fedora-centos-red-hat-rhel/).
```

Comment:
Are you sure it is about Redis and not Postgres? Maybe a typo?
Hmmm 2012 article from an unofficial blog, I'd like to avoid that in a README ;)
If you can find something more official from Postgres github account or website that would be better